core/src/test/java/io/grpc/StatusTest.java

```diff
@@ -49,4 +52,25 @@ public void verifyExceptionMessage() {
     assertEquals("CANCELLED: This is a test",
         Status.CANCELLED.withDescription("This is a test").asException().getMessage());
   }
+
+  @Test
+  public void impossibleCodeValue() {
+    assertEquals(Code.UNKNOWN, Status.fromCodeValue(-1).getCode());
+  }
+
+  @Test
+  public void sameCauseReturnsSelf() {
+    assertSame(Status.CANCELLED, Status.CANCELLED.withCause(null));
+  }
+
+  @Test
+  public void sameDescriptionReturnsSelf() {
+    assertSame(Status.CANCELLED, Status.CANCELLED.withDescription(null));
+    assertSame(Status.CANCELLED, Status.CANCELLED.augmentDescription(null));
+  }
+
+  @Test
+  public void useObjectHashCode() {
+    assertEquals(Status.CANCELLED.hashCode(), System.identityHashCode(Status.CANCELLED));
```

Comment:
@madongfly I looked around Jacoco and there doesn't seem to be a good way to disable coverage for parts of a file.  It hints that there /should/ be a way on their wiki, but there isn't anything yet.  This is noticeable in things like:

private Foo() {} // don't instantiate

That should never be covered, yet jacoco still counts it as uncovered.  I think we decided that upping our code coverage was a P0 OKR, and it seems reasonable to test the lines of code in our library that could reasonably happen.